src/meter/Meter.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import FlexView from '../flex/FlexView';
+import find from 'lodash/collection/find';
+import { warn } from '../utils/log';
+
+const Range = t.refinement(t.struct({
+  startValue: t.maybe(t.Number),
+  endValue: t.Number,
+  fillingColor: t.String,
+  labelColor: t.String
+}), r => r.startValue < r.endValue);
+
+const labelFormatter = (currentValue, startValue, endValue) =>(
```

Comment:
proeslintip: always have the latest `buildo/eslint-config` installed.. it would have caught the missing space after `=>`